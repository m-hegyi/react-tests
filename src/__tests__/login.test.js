import React from "react";
import renderer, { act } from "react-test-renderer";
import Login from "../components/login";

it("Render the login form correctly", () => {
  const component = renderer.create(<Login />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it("Submit login form with valid data", () => {
  const component = renderer.create(<Login />);
});
