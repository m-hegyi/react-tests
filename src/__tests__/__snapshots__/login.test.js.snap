// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the login form correctly 1`] = `
<div>
  <div>
    <label>
      Username
      <input
        onChange={[Function]}
        placeholder="Please provide your username"
        type="text"
        value=""
      />
    </label>
  </div>
  <div>
    <label>
      Password
      <input
        onChange={[Function]}
        placeholder="Please provivde your password"
        type="text"
        value=""
      />
    </label>
  </div>
  <button
    className="btn"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    Login
  </button>
</div>
`;
